test/e2e/specs/all-features/record/related-table-link.spec.js

```diff
@@ -378,7 +378,7 @@ describe('View existing record,', function() {
                     expect(relatedTableLink.isDisplayed()).toBeTruthy();
 
                     browser.wait(EC.elementToBeClickable(relatedTableLink));
-                    relatedTableLink.click().then(function() {
+                    browser.executeScript("return $('#rt-heading-schedule record-action-bar a.more-results-link').click();").then(function() {
```

Comment:
If there is something wrong with the selector that we have defined for this, please fix the selector instead of changing the code. The selector is currently fetching the element with `#rt-heading-schedule .more-results-link` which should be working fine unless the more results link is no longer a child to the "related table heading".